tests/unit/test-util/test.adb.js

```diff
@@ -15,6 +15,7 @@ import ADBUtils from '../../../src/util/adb';
 const fakeADBPackageList = `
 package:org.mozilla.fennec
 package:org.mozilla.firefox
+package:com.some.firefox.fork
```

Comment:
For test coverage of prefix matching vs exact matching, please add another line such as:
`package:com.some.firefox.fork2`

Reply:
I don't think I need to, I'm going to use 2 test cases one with the full package name and assert that it resolves to something, and another with not the full package name and assert that it resolves to nothing

Reply:
oh, I get it now, will do 👍